test/integration/resultsPage.js

```diff
@@ -140,46 +140,31 @@ describe('Results page', () => {
 
   describe('Surgeries with booking system which can be linked to', () => {
     it('should return a booking link for a surgery', (done) => {
-      const search = 'Crookes Valley Medical Centre Sheffield';
+      const search = 'Medi Access';
       const postcode = '';
 
       assertSearchResponse(search, postcode, done, (err, res) => {
         const $ = cheerio.load(res.text);
         const searchResults = $('.results__item--nearby .results__details .results__name a').first();
 
-        expect(searchResults.text()).to.equal('Crookes Valley Medical Centre');
-        expect(searchResults.attr('href')).to.equal('https://systmonline.tpp-uk.com/Login?PracticeId=C88057');
+        expect(searchResults.text()).to.equal('Medi Access Ltd');
+        expect(searchResults.attr('href')).to.equal('https://systmonline.tpp-uk.com/Login?PracticeId=B81693');
       });
     });
   });
 
   describe('Surgeries with booking system which can not be linked to', () => {
-    describe('when the surgery has a website', () => {
```

Comment:
I can't see another surgery where this condition is true so I've removed the test.

Reply:
How about 'Sunlight Centre Surgery'?

Reply:
Great suggestion. How did you find it?